lib/kuikka_website/forum/topic.ex

```diff
@@ -33,10 +33,12 @@ defmodule KuikkaWebsite.Forum.Topic do
   @spec changeset(t, map) :: Ecto.Changeset.t
   def changeset(schema = %__MODULE__{}, params \\ %{}) do
     schema
-    |> cast(params, [:title, :content, :createtime, :modifytime])
+  #  |> cast(params, [:title, :content, :createtime, :modifytime])
+    |> cast(params, [:title, :content])
     |> validate_required([:title, :content])
     |> validate_length(:title, min: 1)
     |> validate_length(:content, min: 1)
+    |> put_assoc(:member, KuikkaWebsite.Member)
```

Comment:
Add assoc function topic module like this
```elixir
defp assoc(changeset, _key, nil), do: changeset
defp assoc(changeset, key, val), do: put_assoc(changeset, key, val)
```

Which can now be called on changeset like this
```elixir
|> assoc(:member, params[:member] || params["member"])
```